ui/admin/app/templates/scopes/scope/roles/role/loading.hbs

```diff
@@ -0,0 +1,9 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<Hds::ApplicationState class='loading-indicator' as |A|>
+  <A.Header @title={{t 'states.loading.title'}} @icon='loading' />
+  <A.Body @text={{t 'states.loading.generic-description'}} />
+</Hds::ApplicationState>
```

Comment:
we preload the scopes in the `scopes.scope.roles.role` route and if there are a lot of scopes then there is an apparent lag. So I added this generic loading state.

Reply:
Should this be in the `roles/loading` or `roles/role/loading`? I forget how the loading hierarchy works, is the roles loading indicator covered by `scopes/scope/loading.hbs` but role isn't?

Reply:
Yeah `roles` is covered by the loading template in the `scope` path tho im unsure why it doesn't work for the `role` route since it should bubble up to find a loading template. 🤷

Great catch, thank you for pointing out the `loading` action to prevent from showing the loading indicator in search/filtering.